tests/Builder/ShowBuilderTest.php

```diff
@@ -175,13 +178,35 @@ public function testFixFieldDescription()
         $fieldDescription->getTemplate()->willReturn(false)->shouldBeCalledTimes(2);
         $fieldDescription->setTemplate(null)->shouldBeCalledTimes(1);
 
-        $fieldDescription->getMappingType()->willReturn(ClassMetadata::ONE_TO_ONE)->shouldBeCalledTimes(2);
+        $fieldDescription->getMappingType()->willReturn($mappingType)->shouldBeCalledTimes(2);
 
-        $fieldDescription->setTemplate('@SonataAdmin/CRUD/Association/show_one_to_one.html.twig')
+        $fieldDescription->setTemplate($template)
             ->shouldBeCalledTimes(1);
 
         $admin->attachAdminClass($fieldDescription)->shouldBeCalledTimes(1);
 
         $this->showBuilder->fixFieldDescription($admin->reveal(), $fieldDescription->reveal());
     }
+
+    public function testFixFieldDescriptionData()
```

Comment:
A data provide that starts with `test`? Is phpunit ok with this?

Reply:
I did not have any errors, so it should be ok but I will rename it as this was a mistake.